Dockerfile

```diff
@@ -1,9 +1,8 @@
-FROM resin/nuc-node:6-20161215
+FROM balenalib/intel-nuc-node:6-stretch-build-20190127
```

Comment:
could use the latest one, maybe?

Reply:
@imrehg `sid` is unstable. `stretch` is marked Debian stable.

The Dockerfile is tagged to specific versions to keep builds functioning.

Reply:
@ZubairLK I mean "tag to a specific one, but more recent" :P And strech is correct.

Reply:
ah. now I see. will bump/test/build/merge.

Reply:
not bumping the base image at this point.
I managed to get a test run to complete with this tag. But autohat doesn't seem to want to start qemu on my laptop anymore (which I think is my laptop/distro). I'll merge this.